src/homeform.cpp

```diff
@@ -1963,9 +1972,14 @@ void homeform::update() {
         pidHR->setValue(QString::number(treadmill_pid_heart_zone));
 
         if (trainProgram) {
-            peloton_offset->setValue(QString::number(trainProgram->offsetElapsedTime()) + QStringLiteral(" sec."));
+            peloton_offset->setValue(trainProgram->remainingTime().toString("h:mm:ss"));
+            peloton_offset->setSecondLine(QString::number(trainProgram->offsetElapsedTime()) + QStringLiteral(" sec."));
             remaningTimeTrainingProgramCurrentRow->setValue(
                 trainProgram->currentRowRemainingTime().toString(QStringLiteral("h:mm:ss")));
+            remaningTimeTrainingProgramCurrentRow->setSecondLine(
+                trainProgram->currentRowElapsedTime().toString(QStringLiteral("h:mm:ss")));
```

Comment:
i think for this we should make a video and show this to the facebook community. I would like to have a feedback from the users because it could change the current behaviour. What do you think?

Reply:
Yeah sorry, now I got your idea behind is!